test/LaTeXImportTests.js

```diff
@@ -0,0 +1,297 @@
+        });
+
+        it('cot, sec, csc', () => {
+            assert.equal('\\cot\\left(x\\right)', nerdamer.convertFromLaTeX('\\cot(x)').toTeX());
+            assert.equal('\\sec\\left(x\\right)', nerdamer.convertFromLaTeX('\\sec(x)').toTeX());
+            assert.equal('\\csc\\left(x\\right)', nerdamer.convertFromLaTeX('\\csc(x)').toTeX());
+        });
+        it('arcsin, arccos, arctan', () => {
+            assert.equal('\\arcsin\\left(x\\right)', nerdamer.convertFromLaTeX('\\arcsin(x)').toTeX());
+            assert.equal('\\arccos\\left(x\\right)', nerdamer.convertFromLaTeX('\\arccos(x)').toTeX());
+            assert.equal('\\arctan\\left(x\\right)', nerdamer.convertFromLaTeX('\\arctan(x)').toTeX());
+        });
+        it('sinh, cosh, tanh', () => {
+            assert.equal('\\sinh\\left(x\\right)', nerdamer.convertFromLaTeX('\\sinh(x)').toTeX());
+            assert.equal('\\cosh\\left(x\\right)', nerdamer.convertFromLaTeX('\\cosh(x)').toTeX());
+            assert.equal('\\tanh\\left(x\\right)', nerdamer.convertFromLaTeX('\\tanh(x)').toTeX());
+        });
+        it('coth, sech, ,csch', () => {
+            assert.equal('\\coth\\left(x\\right)', nerdamer.convertFromLaTeX('\\coth(x)').toTeX());
+            assert.equal('\\sech\\left(x\\right)', nerdamer.convertFromLaTeX('\\sech(x)').toTeX());
+            assert.equal('\\csch\\left(x\\right)', nerdamer.convertFromLaTeX('\\csch(x)').toTeX());
+        });
+        it('sin of pi/2', () => {
+            assert.equal("1", nerdamer.convertFromLaTeX('\\sin(\\pi/2)').toTeX());
+        });
+        it('sin of 90deg', () => {
+            assert.equal("1", nerdamer.convertFromLaTeX('\\sin(90\\deg)').toTeX());
+        });
+    });
+    context('Logs', () => {
+        it('log()', () => {
+            assert.equal("1", nerdamer.convertFromLaTeX('\\log(e)').toTeX());
+        });
+        it('log', () => {
+            assert.equal("1", nerdamer.convertFromLaTeX('\\log e').toTeX());
+        });
+        it('ln', () => {
+            assert.equal("1", nerdamer.convertFromLaTeX('\\ln(e)').toTeX());
+        });
+        it('log_2', () => {
+            assert.equal("1", nerdamer.convertFromLaTeX('\\log_2(2)').toTeX());
+        });
+        it('log_{10}', () => {
+            assert.equal("1", nerdamer.convertFromLaTeX('\\log_{10}(10)').toTeX());
+        });
+    });
+    context('Roots', () => {
+        it('square root', () => {
+            assert.equal("2", nerdamer.convertFromLaTeX('\\sqrt{4}').toTeX());
+            assert.equal("\\sqrt{5}", nerdamer.convertFromLaTeX('\\sqrt{5}').toTeX());
+        });
+        it('cube root', () => {
+            assert.equal("2", nerdamer.convertFromLaTeX('\\sqrt[3]{8}').toTeX());
+            assert.equal("\\sqrt[3]{5}", nerdamer.convertFromLaTeX('\\sqrt[3]{5}').toTeX());
+        });
+        it('nth root', () => {
+            assert.equal("\\sqrt[n]{2", nerdamer.convertFromLaTeX('\\sqrt[n]{2}').toTeX());
```

Comment:
Is a close brace missing?
```suggestion
            assert.equal("\\sqrt[n]{2}", nerdamer.convertFromLaTeX('\\sqrt[n]{2}').toTeX());
```

Reply:
Yes it is!